en/developer-docs/docs/administer/control-access-in-the-choreo-console.md

```diff
@@ -1,49 +1,153 @@
 
-Choreo uses **Roles**, **Groups**, and a **Mapping level** to control access to the Choreo Console as follows:
+## Access Control (Authorization) Concepts
 
-- **Role** : Role is a collection of permissions. Choreo has a predefined set of roles with permissions assigned to them. [Learn more](../choreo-concepts/organization.md#roles)
-- **Group** : Group is a collection of users. A user group requires a role or multiple roles to be assigned to it so that the users in those groups get the relevant permissions via the assigned roles. [Learn more](../choreo-concepts/organization.md#groups)
+To understand how to configure access control in Choreo, it is important to understand how access control is modelled in Choreo.
 
-- **Mapping level** : A mapping level defines the extent at which a role-group mapping can be done. Choreo has two defined resource levels.
-    - **Organization** : You can assign a role to a group or associate a group with a role within the organization. This ensures that
-                         all users in a group inherit the permissions granted by that role across all organizational resources.
-                         For example, if a user has edit_project permission at the organization mapping level, that user can edit all the projects in the organization.
-    - **Project** : You can assign a role to a group or associate a group with a role within a specific project resource. This ensures
-                    that users in the group inherit the permissions granted by that role only within the context of the specified project.
-                    For example, If a user has edit_project permission at the project mapping level, that user can only edit the specified project.
+### Fundamental Question in Access Control
 
+Access control of any system boils down to answering the following question.
 
-In Choreo, authorization operates by assigning a role to a group at a specified level. The level at which the role is assigned determines the extent of permissions granted to users.
+> *Can this **user** perform this **action** on this **resource**?*
 
-!!! warning "Important"
-    Avoid assigning multiple roles to a single user across different projects or levels (organization and project). Such assignments can grant users unintended permission to some projects, allowing them to perform tasks they shouldn't have access to. Therefore, it is recommended to assign only one role to a user across projects or levels to ensure proper access control.
+???+ example
+    Can Harry view logs of development environment of engineering project? 
+    
+    - `Harry` is the **user**  
+    - `view logs` is the **action**  
+    - `development environment of engineering project` is the **resource**   
+
+### Permission
+
+In Choreo access control model, a permission is the right to perform a specific action. 
+
+???+ example
+    A user can configure a custom domain only if the user has `choreo:domain_manage` permission.
```

Comment:
Can we change this to a commonly used permission like - `create component` or something